react-form-controls/my-app/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState, type FormEvent } from 'react';
+
+export default function ControlledForm() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  function handleSubmit(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    const form = new FormData(event.currentTarget);
```

Comment:
This is a controlled form, so all the input values are in your state variables. You don't need to use `FormData` to get them. You can just log the username and password directly.